test/unit/gateways/payu_latam_test.rb

```diff
@@ -24,9 +24,9 @@ def setup
       tax_return_base: 0,
       email: "username@domain.com",
       ip: "127.0.0.1",
-      device_session_id: 'vghs6tvkcle931686k1900o6e1',
-      cookie: 'pt1t38347bs6jc9ruv2ecpv7o2',
-      user_agent: 'Mozilla/5.0 (Windows NT 5.1; rv:18.0) Gecko/20100101 Firefox/18.0',
+      device_session_id: "vghs6tvkcle931686k1900o6e1",
```

Comment:
These lines aren't actually changed, but they appear to be due to quote changes.

Reply:
Yeah I thought I would make the quotes more uniform while I was in here because they were bothering me.

Reply:
That's fair, though two notes on that: first, that should probably be a separate commit, because it'll otherwise confuse `git blame`; and second, I've been told (and tools like RuboCop generally enforce) the quote format to standardize on for strings that do not require interop in Ruby would be `''`, not `""`, specifically so that adding a random `#` doesn't suddenly result in surprising behavior. So I think you'd want to go the other direction here (unless we've got code standards that specify the reverse is the standard within AM).

Reply:
Fair points; Something made me think interops need double instead of single.